modules/test/playwright/tests/export-import-web/main/site.import.spec.ts

```diff
@@ -410,11 +419,259 @@ test('can import a lar file selecting some items to import', async ({
+			site1 = await apiHelpers.headlessSite.createSite({
+				name: getRandomString(),
+				templateKey: 'com.liferay.site.initializer.teaser.showcase',
+				templateType: 'site-initializer',
+			});
+
+			apiHelpers.data.push({id: site1.id, type: 'site'});
+
+			await apiHelpers.objectEntry.postObjectEntry(
+				{
+					textField: getRandomString(),
+					[objectRelationship.body.name]: [
+						{
+							textField: getRandomString(),
+						},
+					],
+				},
+				`c/${objectDefinition1.name.toLowerCase()}s/scopes/${site1.name}`
+			);
+
+			await styleBooksPage.goto(site1.friendlyUrlPath);
+
+			await styleBooksPage.create(getRandomString());
+		});
+
+		await test.step('Export the site 1', async () => {
+			await exportImportPage.goToExport(site1.friendlyUrlPath);
+
+			exportableItems1 = await exportImportPage.getExportableItems();
+
+			expect(exportableItems1.size).toBeGreaterThan(0);
+
+			expect(exportableItems1.has(objectDefinition1.name)).toBe(true);
+
+			expect(exportableItems1.has(objectDefinition2.name)).toBe(true);
+
+			expect(exportableItems1.has('Style Books')).toBe(true);
+
+			exportName = `MyExport-${getRandomString()}`;
+
+			await exportImportPage.exportAll(exportName);
+
+			await expect(
+				exportImportPage.page
+					.locator('[data-qa-id="row"]', {
+						hasText: exportName,
+					})
+					.getByText('Successful')
+			).toBeVisible({timeout: 30000});
+
+			exportFilePath =
+				await exportImportPage.downloadExportProcess(exportName);
+		});
+
+		await test.step('Create the site 2 and import the site 1', async () => {
+			site2 = await apiHelpers.headlessSite.createSite({
+				name: getRandomString(),
+			});
+
+			apiHelpers.data.push({id: site2.id, type: 'site'});
+
+			await exportImportPage.goToExport(site2.friendlyUrlPath);
+
+			exportableItems2 = await exportImportPage.getExportableItems();
+
+			await exportImportPage.goToImport(site2.friendlyUrlPath);
+
+			await exportImportPage.import(exportFilePath);
+
+			await expect(
+					exportImportPage.page
+				.locator('[data-qa-id="row"]', {
+					hasText: exportName,
+				})
+				.getByText('Successful')
+			).toBeVisible({timeout: 60000});
+		});
+
+		await test.step('Assert the exportable items from site 1 and site 2 are equal', async () => {
+			await exportImportPage.goToExport(site2.friendlyUrlPath);
+
+			exportableItems3 = await exportImportPage.getExportableItems();
+
+			expect(exportableItems3.size).toEqual(exportableItems1.size);
+
+			for (const [name, count] of exportableItems1.entries()) {
+
+				// TODO LPD-64899
+
+				if (name === 'Calendar') {
+					continue;
+				}
+				else if (name === 'Pages') {
+					expect(exportableItems3.get(name)).toBe(
+						count + exportableItems2.get('Pages')
+					);
+				}
+				else if (name === 'Style Books') {
+
+					// TODO LPD-64905
+
+					expect(exportableItems3.get(name)).toBeGreaterThanOrEqual(
+						count
+					);
+				}
+				else {
+					expect(exportableItems3.get(name)).toBe(count);
+				}
+			}
```

Comment:
Maybe this should go into a function on a helper instead of here.
It could help to make the test smaller